test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -205,10 +205,11 @@ def host_setup_for_tensorflow_inference(
             hide=True,
         )
     else:
+        # Installing protobuf at 3.20.* to fix errors in using latest protobuf
         ec2_connection.run(
             (
                 f"{python_invoker} -m pip install --user -qq -U 'tensorflow<={framework_version}' "
-                f" 'tensorflow-serving-api<={framework_version}' "
+                f" 'tensorflow-serving-api<={framework_version}' 'protobuf>=3.20,<3.21'"
```

Comment:
This was handled explicitly for TF2.7 on line 193. Seems it affects TF2.6 as well, we can remove the condition on line 193

Reply:
Ah, thanks for adding this!!

Reply:
I'll wait to confirm that all tests pass, and then remove the pip install at line 193.

Reply:
@tejaschumbalkar I have removed the protobuf installation that I have added in the non-graviton case, and made the TF 2.7 specific protobuf installation general to all executions of this test.